packages/react/src/examples/forms/validationStatic.stories.tsx

```diff
@@ -172,28 +127,32 @@ export const Static = () => {
           <div className="hds-example-form__grid-6-6">
             <div className="hds-example-form__item">
               <TextInput
+                {...register('firstName')}
+                onChange={(e) => {
+                  handleChange('firstName', e.currentTarget.value);
```

Comment:
Validation doesn't seem to be working for static examples 🤔 

![form-validation-static](https://user-images.githubusercontent.com/66477579/226285570-7523ba8d-22fa-4e88-8af0-711b2afe6e69.gif)